pkg/codegen/python/doc.go

```diff
@@ -47,16 +47,20 @@ func (d DocLanguageHelper) GetDocLinkForResourceType(pkg *schema.Package, modNam
 
 	var path string
 	var fqdnTypeName string
+	namespace := "pulumi"
+	if pkg.Namespace != "" {
+		namespace = strings.ToLower(strings.ReplaceAll(pkg.Namespace, "_", "-"))
```

Comment:
Why is it `ToLower` here but not in sdk_link? Should we document the format of pkg.Namespace in a comment? Should we extract namespace transformations for all languages in a single utility class?